Makefile

```diff
@@ -93,6 +93,9 @@ TEMPORARY_DIRECTORY = release/.tmp
 APPLICATION_NAME_LOWERCASE = $(shell echo $(APPLICATION_NAME) | tr A-Z a-z)
 APPLICATION_VERSION_DEBIAN = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
 
+# The first (default) rule
+help:
```

Comment:
Can we use `.DEFAULT_GOAL` instead (https://www.gnu.org/software/make/manual/html_node/Special-Variables.html#Special-Variables)?